src/timers.h

```diff
@@ -41,6 +56,8 @@ inline double ChronoClockNow() {
   return FpSeconds(ClockType::now().time_since_epoch()).count();
 }
 
+#endif
```

Comment:
In `ChooseClockType`, do we end up with `std::chrono::high_resolution_clock`,
or `std::chrono::steady_clock`? Does either of them work, or neither works?

Reply:
We end up with `high_resolution_clock`, but neither of them works.